tapdb/sqlc/queries/assets.sql

```diff
@@ -332,7 +332,11 @@ JOIN managed_utxos utxos
                 utxos.lease_expiry <= @now)
            ELSE TRUE
        END
-WHERE spent = FALSE 
+JOIN script_keys
+    ON assets.script_key_id = script_keys.script_key_id
+WHERE spent = FALSE AND 
+        (script_keys.tweaked_script_key <> sqlc.narg('exclude_key') OR
```

Comment:
nit: looks like we're using the equivalent `!=` operator in another place. Since that seems to be widely supported in most modern databases, I actually prefer that because of its same meaning in Go.

Reply:
That's a force of my old SQL habits seeping through right there.